docs/user-interface/handlers/customize.md

```diff
@@ -1,122 +1,316 @@
     {
-        InitializeComponent();
+        public CustomizeEntryPage()
+        {
+            InitializeComponent();
 
+            Microsoft.Maui.Handlers.EntryHandler.EntryMapper["MyCustomization"] = (handler, view) =>
+            {
 #if __ANDROID__
-        Microsoft.Maui.Handlers.ViewHandler.ViewMapper[nameof(IView.Background)] = (h, v) =>
-        {
-            (h.NativeView as Android.Views.View).SetBackgroundColor(Microsoft.Maui.Graphics.Colors.Cyan.ToNative());
-        };
+                handler.NativeView.SetBackgroundColor(Colors.Transparent.ToNative());
+#elif __iOS__
+                handler.NativeView.BorderStyle = UIKit.UITextBorderStyle.None;
+#elif WINDOWS
+                handler.NativeView.FontWeight = Microsoft.UI.Text.FontWeights.Thin;
 #endif
+            };
+        }
     }
 }
 ```
 
-### Remove Android Entry underline
+In this example, the `Entry` customization occurs in a page class. Therefore, all `Entry` controls on Android, iOS, and Windows will be customized once the `CustomizeEntryPage` is displayed. The following customization is performed by using compiler preprocessing directives:
+
+- On Android, the underline is removed from the `Entry`.
+- On iOS, the border is removed from the `Entry`.
+- On Windows, the thickness of the font in the `Entry` is set to thin.
+
+> [!IMPORTANT]
+> The index into a mapper is an arbitrary string and does not have to correspond to a property exposed by a type, or a property on a native control. For example, `["MyCustomization"]` can be specified as a mapper index, with any native control modification being performed as the customization.
+
+## Customize specific control instances
+
+Handlers are global, and customizing a handler for a control will result in all controls of the same type being customized in your app. However, handlers for specific control instances can be customized by subclassing the control, and then by modifying the handler for the base control type only when the control is of the subclassed type. For example, to customize a specific `Entry` control on a page that contains multiple `Entry` controls, you should first subclass the `Entry` control:
+
+```csharp
+using Microsoft.Maui.Controls;
+
+namespace HandlersDemos.Controls
+{
+    public class MyEntry : Entry
+    {
+    }
+}
+```
+
+You can then customize the `EntryHandler`, via its mapper, to perform the desired modification to `MyEntry` instances:
+
+```csharp
+Microsoft.Maui.Handlers.EntryHandler.EntryMapper["MyCustomization"] = (handler, view) =>
+{
+    if (view is MyEntry)
+    {
+#if __ANDROID__
+        handler.NativeView.SetBackgroundColor(Colors.Transparent.ToNative());
+#elif __iOS__
+        handler.NativeView.BorderStyle = UIKit.UITextBorderStyle.None;
+#elif WINDOWS
+        handler.NativeView.FontWeight = Microsoft.UI.Text.FontWeights.Thin;
+#endif
+    }
+};
+```
+
+Any `MyEntry` instances in the app will then be customized as per the handler modification.
```

Comment:
One thing I'm not clear about, do handlers affect already created controls? Or just apply to any control created after the handler is altered?